sumologic/resource_sumologic_cse_network_block_test.go

```diff
@@ -112,3 +114,12 @@ func testCheckNetworkBlockValues(networkBlock *CSENetworkBlock, nAddressBlock st
 		return nil
 	}
 }
+
+func generateRandomCIDRBlock() string {
+	ip := make(net.IP, 4)
```

Comment:
Shouldn't the bits after the prefix (i.e., the last 6) be zeroes?

Reply:
Makes sense, have set last octet 0.